CRM/Batch/BAO/EntityBatch.php

```diff
@@ -17,23 +17,11 @@
 class CRM_Batch_BAO_EntityBatch extends CRM_Batch_DAO_EntityBatch {
 
   /**
-   * Create entity batch entry.
```

Comment:
Can we put back these summary sentences. The drupal standard includes them

Reply:
Yes done.